Automattic/wp-calypso
# Add a new profile page under /profile/:site

Fixes https://github.com/Automattic/dotcom-forge/issues/7973

## Proposed Changes

* Created /profile with the Color Schema selector to manage per-site per-user under the flag `layout/site-level-user-profile`

![image](https://github.com/Automattic/wp-calypso/assets/402286/ea7c39f5-6667-4000-96aa-d493ebb21cec)

#### TO DO
- [ ] When changing the color, it updates the body class but not removes the old one

## Why are these changes being made?


* pfsHM7-1hH-p2

## Testing Instructions

* Go to /profile/:site with the flag `flags=layout/site-level-user-profile`
* Select another color scheme
* Check if reloading the color is the selected
* Go to a `/wp-admin` and check if the selected color match
* Without the flag, the color you see and select is the global color
* Check if nothing is broken


Follow up items:

- [ ] Support doc updates (for the Learn more link up there)
- [ ] Color scheme switcher needs to talk to the new site-level APIs
- [ ] /me/account color scheme area needs a text to say the option has moved


## Pre-merge Checklist
